pkg/controller/migration/controller.go

```diff
@@ -84,7 +85,7 @@ func Add(mgr manager.Manager) error {
 	}
 
 	// Gather migration metrics
-	recordMetrics(mgr.GetClient())
```

Comment:
since you removed this. did you check if now you need the permissions and `monitor.yaml` within the operator(`operator/config/prometheus/monitor.yaml` and within `operator/config/rbac/leader_election_role.yaml`)?

Reply:
I didn't removed it, I just moved it to the monitor directory to have all the monitoring related things organized in the same place, but any way the yaml you referring is not related to the controller metrics (i'm not sure for what it used), but it configured on a different namespace and port. i'm creating the relevant file for the metric recording in the controller addition.